whisper/Dockerfile.ctranslate2

```diff
@@ -1,5 +1,5 @@
 FROM ghcr.io/opennmt/ctranslate2:latest-ubuntu20.04-cuda11.2
-LABEL maintainer="jlouradour@linagora.com"
+LABEL maintainer="hello@linto.ai, jlouradour@linagora.com, dgaynullin@linagora.com"
```

Comment:
@damienlaine Should we use support@linto.ai instead of hello@linto.ai here?

Reply:
Not sure if support is routed correctly, 
contact@linto.ai is the best